azure-pipelines.yml

```diff
@@ -640,34 +640,7 @@ jobs:
       ArtifactName: rpm-mariner$(tag)
 
 
-- job: BuildRpmPackageCentOS7
-  displayName: Build Rpm Package CentOS 7
-  # Uncomment this line to disable this job on Pull Requests
-  # condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual', 'Schedule'))
-  pool:
-    vmImage: 'ubuntu-20.04'
-  steps:
-  - task: Bash@3
-    displayName: 'Build Rpm Package'
-    inputs:
-      targetType: 'filePath'
-      filePath: scripts/release/rpm/pipeline.sh
-    env:
-      IMAGE: centos7
-  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
-    displayName: 'SBOM'
-    inputs:
-      BuildDropPath: $(Build.ArtifactStagingDirectory)
-  - task: PublishPipelineArtifact@0
-    displayName: 'Publish Artifact: rpm'
-    inputs:
-      TargetPath: $(Build.ArtifactStagingDirectory)
-      # Should be 'rpm-centos7', but we keep 'yum' for backward compatibility
-      ArtifactName: yum
-
 # TODO: rpmbuild on Red Hat UBI 8 is slow for unknown reason. Still working with Red Hat to investigate.
```

Comment:
Good news! `BuildRpmPackageUbi8` is not as slow as before. Now it only takes ~20min.

![image](https://user-images.githubusercontent.com/4003950/177119841-42cfc652-2d72-49f6-bbaf-87667b8fc278.png)